packages/zoe/test/unitTests/contracts/test-coveredCall.js

```diff
@@ -142,7 +167,7 @@ test('zoe - coveredCall', async t => {
           );
 
         await E(seat)
-          .getPayout('StrikePrice')
+          .getPayout('StrikePrice1')
```

Comment:
Is this testing that you can use arbitrary named?  or is it that you must use a different name for some underlying reason?

Reply:
Arbitrarily named